pkg/kubelet/kuberuntime/helpers.go

```diff
@@ -202,8 +202,37 @@ func toKubeRuntimeStatus(status *runtimeapi.RuntimeStatus) *kubecontainer.Runtim
 	return &kubecontainer.RuntimeStatus{Conditions: conditions}
 }
 
+func (m *kubeGenericRuntimeManager) getSeccompProfile(annotations map[string]string, containerName string,
+	podSecContext *v1.PodSecurityContext, containerSecContext *v1.SecurityContext) string {
+	fieldProfile := func(scmp *v1.SeccompProfile) string {
+		if scmp.Type == v1.SeccompProfileUnconfined {
+			return "unconfined"
+		}
+		if scmp.Type == v1.SeccompProfileTypeDefault {
+			return v1.SeccompProfileRuntimeDefault
+		}
+		if scmp.Type == v1.SeccompProfileLocalhost && scmp.LocalhostProfile != nil {
+			fname := filepath.Join(m.seccompProfileRoot, *scmp.LocalhostProfile)
+			return "localhost/" + fname
```

Comment:
I think we should make `localhost/` a constant somewhere. We have the same for AppArmor and we will need it in various places.

Reply:
Agreed, just updated accordingly.